types/validator_set.go

```diff
@@ -270,34 +278,21 @@ func (vals *ValidatorSet) VerifyCommit(chainID string, blockID BlockID, height i
 	}
 
 	talliedVotingPower := int64(0)
-	round := commit.Round()
+	commit.SetAddresses(vals.GetAddresses())
```

Comment:
This one looks right, though we don't actually need it here because the only thing we do with the Vote is get the SignBytes